src/components/textarea/textarea.spec.tsx

```diff
@@ -711,12 +711,34 @@ describe("componentWillUnmount", () => {
     });
   });
 
-  it("renders with the expected border radius styling", () => {
+  it("renders with the expected default border radius styling", () => {
     assertStyleMatch(
       {
         borderRadius: "var(--borderRadius050)",
       },
       mount(<Textarea />).find(StyledInputPresentation)
     );
   });
+
+  it("renders with the expected custom border radius styling", () => {
+    assertStyleMatch(
+      {
+        borderRadius: "16px 8px 24px 36px",
+      },
+      mount(<Textarea borderRadius="16px 8px 24px 36px" />).find(
+        StyledInputPresentation
+      )
+    );
+  });
+
+  it("renders with the expected custom border radius styling using a design-token value", () => {
```

Comment:
**comment** (non-blocking): I'm not convinced we need 2 separate tests here, since it's a simple string prop that just gets passed straight down to the styles with no messing about. But I don't object to explicitly checking either, it's always better to have too many tests than too few.

Reply:
My thoughts here were that if we upgrade the `design-tokens` package at any point, we at least have a unit test to test if any issues were to appear before it gets to the Chromatic stage. I'm easy though. I can remove the test if you think I'm being overly cautious.

Reply:
I'm not sure upgrading `design-tokens` would cause any issues that this test would catch (as the token value wouldn't actually change, just the styles that got rendered by it) - but still, let's keep it as there's no such thing really as "overly cautious" when it comes to tests.